geomesa-features/geomesa-feature-common/src/main/scala/org/locationtech/geomesa/features/serialization/TwkbSerialization.scala

```diff
@@ -48,7 +51,7 @@ trait TwkbSerialization[T <: NumericWriter, V <: NumericReader]
         val coord = geometry.getCoordinate
         // check for dimensions - use NaN != NaN to verify z coordinate
```

Comment:
Update comment?

Reply:
oops, sorry didn't see this. I think it's still somewhat valid as that's what `Double.isNaN` does anyway